examples/shell/openiotsdk/CMakeLists.txt

```diff
@@ -23,7 +23,7 @@ get_filename_component(SHELL_COMMON ${CHIP_ROOT}/examples/shell/shell_common REA
 
 list(APPEND CMAKE_MODULE_PATH ${OPEN_IOT_SDK_CONFIG}/cmake)
 
-set(APP_TARGET chip-openiotsdk-shell-example)
+set(APP_TARGET chip-openiotsdk-shell-example_ns)
```

Comment:
Do we need to rename this target ?

Reply:
I think, yes. Then we remain in TF-M apps convention I think. Also, this gives a clearer division of the application. `chip-openiotsdk-shell-example_ns.elf` is the Matter example and `chip-openiotsdk-shell-example.elf` is a merged set of apps that could be loaded into the device. I also use the `chip-openiotsdk-shell-example_ns.elf` for loading symbols in debug session.